docs/adding_packages/README.md

```diff
@@ -110,24 +80,16 @@ The specific steps to submitting changes are:
 * Build and test the new recipe in several combinations. Check [developing recipes](../developing_recipes_locally.md) for tips.
 * [Commit and push to your fork repository](https://docs.github.com/en/get-started/using-git/pushing-commits-to-a-remote-repository) then
   [submit a pull request](https://github.com/conan-io/conan-center-index/compare).
-* Our automated [build service](#the-build-service) will build 100+ different configurations, and provide messages that indicate if there
-  were any issues found during the pull request on GitHub.
 
 When the pull request is [reviewed and merged](../review_process.md), those packages are published to [JFrog's ConanCenter](https://conan.io/center/)
 and are made available for everyone.
 
 ## The Build Service
 
 The **build service** associated to this repository will generate binary packages automatically for the most common platforms and compilers.
-See [the Supported Platforms and Configurations page](../supported_platforms_and_configurations.md) for a list of generated configurations.
-For a C++ library, the system is currently generating more than 100 binary packages.
+For a C++ library, the system is currently generating more than 30 binary packages.
```

Comment:
See my comment above - this is going to change soon, I think it's best to keep the supported platforms md for now

Reply:
Addressed by 75461cc13016357ac2b7a51d7e2c5b060afa05af